modules/gdscript/tests/scripts/parser/features/export_variable.gd

```diff
@@ -20,6 +22,10 @@ const Utils = preload("../../utils.notest.gd")
 @export var test_image: Image
 @export var test_timer: Timer
 
+# Global custom classes
```

Comment:
```suggestion
# Global custom classes.
```

Reply:
While addressing this, please also squash the commits so there's only one commit in this PR. See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

Reply:
Ah just saw that extra commit sorry, will do.